### ISSUE TITLE
Not working with Expo

### ISSUE THREAD

**Issue by deadcoder0904** [shoutem/ui#324]
Using SDK 22.0 of Expo & this does not work

Throws error 

```
undefined is not an object (evaluating '_react.PropTypes.shape')
```

**Comment by abelta** [comment]
Most probably it's an incompatibility with newest React version. Has that same problem right now and fixed it reverting to `react@16.0.0-alpha.12`.

Even with an older version I keep getting warnings like these:
```
Warning: PropTypes has been moved to a separate package.
Warning: Native component for "PhotoViewAndroid" does not existWarning: Native component for "BVLinearGradient" does not exist
Warning: React.createClass is no longer supported.
```

It's like the library hasn't been updated in months.

**Comment by deadcoder0904** [comment]
Yep, thats what makes me sad. Nothing is updated at @shoutem for months. But, what can we say its open source. I'm not using this now since I tried it many times.

**Comment by outaTiME** [comment]
yup same here =(

<img width="537" alt="screen shot 2017-11-01 at 15 47 53" src="https://user-images.githubusercontent.com/140472/32291513-0e574a90-bf1c-11e7-8b3a-399dfa44fcb3.png">

**Comment by chris-rowe** [comment]
I manually went through and changed and the propType code and it tool quite a while.  Then I started getting the React.createClass errors. I started changing that code and just decided to give up because the errors were never ending.  I created two issues for these problems.

**Comment by deadcoder0904** [comment]
I don't think this is gonna be up-to-date anymore so my advice is either use React Native Elements or Nativebase or Custom because Shoutem people's business is working on Old RN versions so I don't think they will be working on this. 

But, they did make great philosophies like [@shoutem/theme](https://github.com/shoutem/theme) & [@shoutem/animation](https://github.com/shoutem/animation). Not original, but awesome.

**Comment by chris-rowe** [comment]
I just installed shoutem/ui, then ran the react codemod tool on everything in shoutem/ui and shoutem/theme. Then a handful of other packages are doing the same thing. Work your way through those with codemod as well. Then the last error you should get should complain about unknown font Rubik. Run react-native link, restart the packager and it worked like a charm.

**Comment by caiotarifa** [comment]
Same here. Any luck with this?

**Comment by Unforgiven-wanda** [comment]
@chris-rowe Care to create a fork with these changes?

**Comment by Vladimir-Vdovic** [comment]
The PropTypes issue has been resolved with [version 0.22.0](https://www.npmjs.com/package/@shoutem/ui), along with a fix for Linear Gradient which was causing issues for `create-react-native-apps`.
